app/src/main/java/org/mozilla/focus/utils/Features.kt

```diff
@@ -16,7 +16,7 @@ object Features {
     /**
      * Erase tabs CFR
      */
-    var IS_ERASE_CFR_ENABLED = AppConstants.isDevOrNightlyBuild
+    val IS_ERASE_CFR_ENABLED = AppConstants.isDevOrNightlyBuild
```

Comment:
Just wanted to commend this change.
Great that a feature depending on the build type is not a `var` anymore.